### DISCUSSION TITLE
my name is too short. (south korea general name)

### DISCUSSION THREAD

**Discussion by chan-gyo** [community/community#136346] [GitHub Education]
### Hi everyone,

my legal name is 정찬교. first name is 찬교, last name is 정. so I cannot upload billing information. what should I do??

**Comment by mhaemnn** [comment]
I'm confused about what your problem is regarding this

**Comment by Altamash-dev** [comment]
### Hi There,
            If it's a problem with the number of characters in your name I suggest you to enter your name in English considering that the Legal Name* matches with your Transcripts/Proof Of Enrollment provided by your Institute.
